diff --git a/dwm.c b/dwm.c
index 36191f8..dcd2016 100644
--- a/dwm.c
+++ b/dwm.c
@@ -59,7 +59,7 @@
 
 /* enums */
 enum { CurNormal, CurResize, CurMove, CurLast }; /* cursor */
-enum { SchemeNorm, SchemeSel, SchemeStxt }; /* color schemes */
+enum { SchemeNorm, SchemeSel, SchemeLtsym, SchemeStxt }; /* color schemes */
 enum { NetSupported, NetWMName, NetWMState, NetWMCheck,
        NetWMFullscreen, NetActiveWindow, NetWMWindowType,
        NetWMWindowTypeDialog, NetClientList, NetLast }; /* EWMH atoms */
@@ -243,7 +243,7 @@ static char stext[256];
 static char *stoks[16];
 static int screen;
 static int sw, sh;           /* X display screen geometry width, height */
-static int bh, blw = 0;      /* bar geometry */
+static int bh;               /* bar geometry */
 static int lrpad;            /* sum of left and right padding for text */
 static int (*xerrorxlib)(Display *, XErrorEvent *);
 static unsigned int numlockmask = 0;
@@ -720,18 +720,22 @@ dirtomon(int dir)
 void
 drawbar(Monitor *m)
 {
-	int x, w, tx, tw = 0;
+	int x, w, tx, tw = 0, lw;
 	int boxs = drw->fonts->h / 9;
 	int boxw = drw->fonts->h / 6 + 2;
 	unsigned int i, occ = 0, urg = 0;
 	Client *c;
 
+	lw = TEXTW(m->ltsymbol) - lrpad + 2; /* 2px right padding */
+	drw_setscheme(drw, scheme[SchemeLtsym]);
+	drw_text(drw, m->ww - lw, 0, lw, bh, 0, m->ltsymbol, 0);
+
 	/* draw status first so it can be overdrawn by tags later */
 	if (m == selmon) { /* status is only drawn on selected monitor */
 		for (i = 0; stoks[i]; tw += TEXTW(stoks[i++]) - lrpad);
 		for (i = 0, tx = 0; stoks[i] && i < LENGTH(colors); tx += TEXTW(stoks[i++]) - lrpad) {
 			drw_setscheme(drw, scheme[i & 1 ? SchemeNorm : SchemeStxt + i / 2]);
-			drw_text(drw, m->ww - tw + tx, 0, TEXTW(stoks[i]) - lrpad, bh, 0, stoks[i], 0);
+			drw_text(drw, m->ww - lw - tw + tx, 0, TEXTW(stoks[i]) - lrpad, bh, 0, stoks[i], 0);
 		}
 	}
 
@@ -751,11 +755,8 @@ drawbar(Monitor *m)
 				urg & 1 << i);
 		x += w;
 	}
-	w = blw = TEXTW(m->ltsymbol);
-	drw_setscheme(drw, scheme[SchemeNorm]);
-	x = drw_text(drw, x, 0, w, bh, lrpad / 2, m->ltsymbol, 0);
 
-	if ((w = m->ww - tw - x) > bh) {
+	if ((w = m->ww - lw - tw - x) > bh) {
 			drw_setscheme(drw, scheme[SchemeNorm]);
 			drw_rect(drw, x, 0, w, bh, 1, 1);
 	}
@@ -1132,14 +1133,8 @@ maprequest(XEvent *e)
 void
 monocle(Monitor *m)
 {
-	unsigned int n = 0;
 	Client *c;
 
-	for (c = m->clients; c; c = c->next)
-		if (ISVISIBLE(c))
-			n++;
-	if (n > 0) /* override layout symbol */
-		snprintf(m->ltsymbol, sizeof m->ltsymbol, "[%d]", n);
 	for (c = nexttiled(m->clients); c; c = nexttiled(c->next))
 		resize(c, m->wx, m->wy, m->ww - 2 * c->bw, m->wh - 2 * c->bw, 0);
 }
